cache/lru_cache.h

```diff
@@ -144,8 +150,13 @@ struct LRUHandle {
   void SetPriority(Cache::Priority priority) {
     if (priority == Cache::Priority::HIGH) {
       flags |= IS_HIGH_PRI;
+      flags &= ~IS_LOW_PRI;
+    } else if (priority == Cache::Priority::LOW) {
+      flags |= IS_LOW_PRI;
+      flags &= ~IS_HIGH_PRI;
```

Comment:
Maybe switch the order of these two statements so high is set before low on all three branches

Reply:
Done, Thanks.